src/Files.App/Views/Layouts/ColumnLayoutPage.xaml

```diff
@@ -199,12 +201,6 @@
 					SelectionMode="Extended"
 					Tapped="FileList_ItemTapped">
 
-					<ListView.ItemContainerTransitions>
```

Comment:
This is a bit unrelated but it makes the Column View consistent with the Grid View in this regard.